modules/database/src/ioc/dbStatic/dbFldTypes.h

```diff
@@ -87,6 +101,7 @@ mapdbfType pamapdbfType[DBF_NTYPES] = {
 #define DBR_ENUM        DBF_ENUM
 #define DBR_PUT_ACKT    DBR_ENUM+1
 #define DBR_PUT_ACKS    DBR_PUT_ACKT+1
+#define DBR_VFIELD      (DBR_PUT_ACKS+1)
```

Comment:
How about `DBR_QFIELD`, `QField`, `get_qfield()`, `put_qfield()` etc.? You already own the letter Q in the EPICS world (and it has a bigger Scrabble score)…

Reply:
Wow.  1/26th of the alphabet.  Such an honor!